genesis/ext/pyrender/interaction/aabb.py

```diff
@@ -79,3 +72,33 @@ def from_center_and_size(cls, center: Vec3, size: Vec3) -> 'AABB':
         max = center + 0.5 * size
         bounds = np.stack((min.v, max.v))
         return cls(bounds)
+
+
+class OBB(AABB):
+    pose: Pose
+
+    def __init__(self, v: NDArray[np.float32], pose: Pose):
+        super().__init__(v)
+        self.pose = pose
+
+    def raycast(self, ray: Ray) -> RayHit:
+        inv_pose = self.pose.get_inverse()
```

Comment:
Do not compute inverse explicit. This is slow. You should rather implement `inv_transform_point`, `inv_transform_direction`